templates/alerts.html

```diff
@@ -32,44 +32,44 @@
                 cell.setAttribute('field_name', fields[i]);
                 var value = alert[test];
                 if (fields[i] == "reference") {
-                    value = "<p><a href=\"" + alert['graphurl'] + "\">" + 'graphurl' + "</a></p>"+ "<p><a href=\"" + alert['changeset'] + "\">" + 'changeset' + "</a></p>";
+                    value = "<p><a href=\"" + alert['graphurl'] + "\">" + 'graphurl' + "</a>"+ "  ||  <a href=\"" + alert['changeset'] + "\">" + 'changeset' + "</a></p>";
                     cell.onclick = "";
 
                 } else if (fields[i] == "status") {
-                    value = "<select class='form-control' id=\"" + alert["id"] + "-status\" onChange=\"updateStatus(" + alert['id'] + ", '" + alert['duplicate'] + "', '" + alert['bug'] + "', '" + alert['mergedfrom'] + "');\">";
-                    var options = ["NEW", "Back Filling", "Investigating", "Duplicate", "Resolved", "Shipped", "Wont Fix", "False Alarm", "Ignore", "Not Tracking", "Backout", "Too Low"];
-                    for (var j = 0; j < options.length; j++) {
-                        value += "<option ";
-                        if (alert['status'] == options[j]) {
-                            value += "selected";
-                        }
-                        value += ">" + options[j] + "</option>";
-                    }
-                    value += "</select>";
-                    cell.onclick = "";
+                    // value = "<select class='form-control' id=\"" + alert["id"] + "-status\" onChange=\"updateStatus(" + alert['id'] + ", '" + alert['duplicate'] + "', '" + alert['bug'] + "', '" + alert['mergedfrom'] + "');\">";
+                    // var options = ["NEW", "Back Filling", "Investigating", "Duplicate", "Resolved", "Shipped", "Wont Fix", "False Alarm", "Ignore", "Not Tracking", "Backout", "Too Low"];
+                    // for (var j = 0; j < options.length; j++) {
+                    //     value += "<option ";
+                    //     if (alert['status'] == options[j]) {
+                    //         value += "selected";
+                    //     }
+                    //     value += ">" + options[j] + "</option>";
+                    // }
+                    // value += "</select>";
+                    // cell.onclick = "";
```

Comment:
we should still print the status, or maybe color code the row- somehow we need to determine the status.